packages/fxa-auth-server/fxa-oauth-server/scripts/sops-key-config.js

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+
+ Usage:
+ scripts/sops-key-config.js [extract|insert] path/to/encrypted/config.yaml
+
+ This script can be used to manage OpenID key configuration in a config file encrypted
+ via SOPS [1], like we use for production deploys.
+
+ When `extract` is specified, this will extract the OpenID `key`, `newKey` and `oldKey`
+ config entries from the specified SOPS-encrypted config file, and write them into the
+ files specified by `keyFile`, `newKeyFile` and `oldKeyFile` where they can be operated
+ on locally.
+
+ When `insert` is specified, this will write the contents of `keyFile`, `newKeyFile` and
+ `oldKeyFile` into the specified SOPS-encrypted config file, where they can be committed
+ to the production config store.
+
+ [1] https://github.com/mozilla/sops
+
+ */
+
+//eslint-disable no-console
+
+const fs = require('fs');
+const assert = require('assert');
+const config = require('../lib/config');
+const yaml = require('js-yaml');
+const { execFileSync } = require('child_process');
+
+const SOPS_ITEM_KEY = 'openid.key';
+const SOPS_ITEM_NEWKEY = 'openid.newKey';
+const SOPS_ITEM_OLDKEY = 'openid.oldKey';
```

Comment:
I don't know what the actual tree paths of these config items are in our actual config file. @jbuck?

Reply:
we're currently using `fxa_oauth::oauth_openid_key` for `SOPS_ITEM_KEY` - I guess we could use `fxa_oauth::oauth_openid_new_key` and `fxa_oauth::oauth_openid_old_key` for the equivalents?